Samples/GdkPixbuf/TestMemoryLeaks/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.IO;
+using GdkPixbuf;
+
+namespace TestMemoryLeaks
```

Comment:
What's the correct/intended way of running this demo? I can run it well enough using the VS debugger, but it would be worth adding instructions.

The VS debugger also seems to fluctuate quite a bit, so I'm not sure how reliable those readings are.

Reply:
I read that using the debugger for this is just "wrong" as it influences the garbage collection. The same goes for compiling and / or running it in debug mode.

For me it is like the memory is more or less stable no visible sawtooth or s.th. similar if i run / compile it in release mode.

Do you have some recommendations regarding the readings? This is not really a sample for the end user. More like internal just for us to verify.